proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,394 @@
+syntax = "proto3";
+
+package redpanda.api.dataplane.v1alpha2;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message MountTopicsRequest {
+  // TopicMount defines the migration of a topic from the cloud storage into this cluster,
+  // so that it becomes available via the Kafka API.
+  message TopicMount {
+    // SourceTopic is the topic name or full reference we want to mount. The full reference
+    // must be used in case the same topic exists more than once. This may be the case if
+    // the same topic has been unmounted multiple times. List all mountable topics to
+    // find the full reference (contains topic name, cluster uuid and revision).
```

Comment:
```suggestion
    // find the full reference (contains topic name, cluster UUID, and revision).
```
Will users also need to use the format `<topic-name>/<cluster-uuid>/<initial-revision>` as with the admin API? Would be good to specify in the field description... we could provide an example as well.

Reply:
Yes they would, they would retrieve the reference in the right format by listing the mountable topics. They wouldn't construct the reference themselves